tests/benchdnn/inputs/graph/pattern/harness_f32_all

```diff
@@ -134,6 +134,31 @@
 --reset --in-shapes=0:384x4096+1:4096x1024+2:384x1 --case=pattern/f32/matmul_post_ops_swish_fusion.json
 --reset --in-shapes=0:384x4096+1:4096x1024+2:1024 --case=pattern/f32/matmul_post_ops_swish_fusion.json
 --reset --case=pattern/f32/matmul_select.json
+# matmul broadcasting tests
+# Basic matmul: standard 2D x 2D case
+--reset --in-shapes=0:128x256+1:256x512 --case=op/f32/matmul.json
```

Comment:
This file pattern/harness_f32_all is for pattern tests. Move these single matmul tests to op/harness_{f32|bf16|f16}_all?

Reply:
good catch! fixed.